spec/jobs/nb_person_push_job_spec.rb

```diff
@@ -5,7 +5,16 @@
     it "calls NationBuilder with correct args" do
       args = { attributes: { email: "user@example.com", phone: '510-555-9999' } }
       expect(Integration::NationBuilder).to receive(:create_or_update_person).with(args)
-      NbPersonPushJob.new.perform("email"=>"user@example.com", "phone"=>"510-555-9999")
+      NbPersonPushJob.new.perform(email: "user@example.com", phone: "510-555-9999")
+    end
+
+    it "creates RSVP when event_id is present" do
+      args = { attributes: { email: "user@example.com" } }
+      expect(Integration::NationBuilder).to receive(:create_or_update_person).
+        with(args) { { 'id' => 6 } }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.